tests/unit/testsuite/Bolt/Boltpay/Model/BoltOrderTest.php

```diff
@@ -331,4 +331,35 @@ public function testCorrectBillingAddressWithNoBillingAddress() {
 
         $this->assertEquals($expected, $result);
     }
-}
\ No newline at end of file
+
+    /**
+     * @test
+     * @group Discount
+     * @dataProvider addDiscountsCases
+     */
+    public function addDiscounts($data = [])
```

Comment:
Please add method comment as to what this is used for.  It appears to be an internal method, so would better be set to private.  If it is a utility method for wider use than this test, then it is best placed in a helper class.